### ISSUE TITLE
Vanilla SWC Test Script is confusing

### ISSUE THREAD

**Issue by bjoyce3** [UA-Carpentries-Workshops/2019-02-23-WorkshopResources#30]
The swc-installation-test-2.py test script that SWC provides is confusing for our attendees. We should create a streamlined version specific to our lessons and remove anything superfluous. The stream of 'failed' messages really confuses people (rightfully so). 

Examples: 
- Test script includes things like Mercurial (we don't use this) which fails the test script (because we don't use it)
- EasyMercurial (fails)
- make fails on some computers (I'm unclear whether that's important or not)

Output from my run
```check command line shell (virtual-shell)...	pass
check text/code editor (virtual-editor)...	pass
check web browser (virtual-browser)...	pass
check Git (git)...	pass
check Mercurial (hg)...	fail
check EasyMercurial...	fail
check make...	pass
check PyPI installer (virtual-pypi-installer)...	pass
check Setuptools (setuptools)...	pass
check Nose (nosetests)...	pass
check Nose Python package (nose)...	pass
check py.test...	pass
check pytest Python package (pytest)...	pass
check SQLite 3 (sqlite3)...	pass
check SQLite Python package (sqlite3-python)...	pass
check Python version (python)...	pass
check IPython script (ipython)...	pass
check IPython Python package (IPython)...	pass
check Argparse (argparse)...	pass
check NumPy (numpy)...	pass
check SciPy (scipy)...	pass
check Matplotlib (matplotlib)...	pass
check Pandas (pandas)...	pass```

**Comment by uhilgert** [comment]
Agreed, Blake. I find the trouble-shooting link even more useful than the test instructions. Any chance you could take a first stab at it? Could we have a separate test script for each of the components? In the extreme one each for gitbash, git, nano, python, Jupyter Notebook, R, and R-Studio?

Thank you.

Uwe

------------------------------------------------------------------------
Uwe Hilgert, Ph.D.

Associate Research Professor
Director of Industry Relations, STEM Training & Workforce Development
BIO5 Institute, The University of Arizona
(O) 520-626-1367  |  (F) 520-626-4824
(E) hilgert@email.arizona.edu<mailto:hilgert@email.arizona.edu>   |  (W) http://www.bio5.org<http://www.bio5.org/>


________________________________

**Comment by bjoyce3** [comment]
Sure thing. I just wanted to document as we go. I'll worry about fixing issues after the weekend :)

**Comment by MarneeDear** [comment]
We can work together on this. I have ideas on how we can do this. I would also like to start a standard repo from which all other repos will derive. We can put setup and test scripts in there and use them as reference on the workshop websites.
